packages/sync/README.md

```diff
@@ -17,3 +17,140 @@ To enable debug please execute in console
 Some certain features can be enabled separately
 
 `localStorage.debug = 'AeroGearSync:OfflineMutations*'`
+
+# Importing the package
+```javascript
+import {
+  createClient,
+  strategies
+} from '@aerogear/datasync-js';
+```
+
+# Configuration
+To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `DataSyncConfig` interface from the SDK.
+
+```javascript
+declare var window: any;
+
+let config: DataSyncConfig = {
+  httpUrl: "my-custom-url",
+  wsUrl: "my-custom-ws-url",
+  dataIdFromObject: "id",
+  storage: window.localStorage,
+  conflictStrategy: strategies.diffMergeClientWins,
+  customLinkBuilder: customLinkBuilder,
+  networkStatus: customNetworkStatus,
+  mutationsQueueName: "offline-mutations-queue",
+  squashing: true
+}
+```
+
+Config | Description | Default
+------ | ------ | ------
+httpUrl | The URL of your http server | N/A
+wsUrl | The URL of your websocket | N/A
+dataIdFromObject | The name of the field to be used as ID | "id"
+storage | The storage you want your client to use | window.localStorage
+conflictStrategy | The conflict resolution strategy your client should use | N/A
+customLinkBuilder | Enables providing custom Apollo Link for processing requests | See `LinksBuilder`
+networkStatus | Implementation of `NetworkStatus` Interface | See `WebNetworkStatus` and `CordovaNetworkStatus`
+mutationsQueueName | The name to store requests under in your offline queue | "offline-mutation-store"
+squashing | Whether or not you wish to squash mutations in your queue | true
+
+
+# Creating a Client
+```javascript
+let client = createClient(config);
+```
+
+
+## Conflict strategies
+You can provide your custom conflict resolution strategies to the client in the config by using the provided `ConflictResolutionStrategy` type from the SDK. An example custom strategy is shown below.
+
+```javascript
+let clientWins = (serverData, clientData) => {
+  return Object.assign(server, client);
+};
+```
+
+To use this strategy pass this function as conflictStrategy in your config object like so:
```

Comment:
Should we also say we provide some strategies out of the box and how to use this?

Reply:
Not now. We should document only offline stuff for the moment.